source/common/secret/sds_api.cc

```diff
@@ -30,11 +30,11 @@ SdsApi::SdsApi(envoy::config::core::v3::ConfigSource sds_config, absl::string_vi
   // This has to happen here (rather than in initialize()) as it can throw exceptions.
   subscription_ = subscription_factory_.subscriptionFromConfigSource(
       sds_config_, Grpc::Common::typeUrl(resource_name), stats_, *this, resource_decoder_);
+
   // TODO(JimmyCYJ): Implement chained_init_manager, so that multiple init_manager
```

Comment:
Is this TODO still relevant?

Reply:
I think so; Added a note where to look re: handling of init targets